src/applications/ivc-champva/10-7959C/containers/ConfirmationPage.jsx

```diff
@@ -80,7 +80,6 @@ export function ConfirmationPage(props) {
     showMail: true,
     allPages: form.pages,
     fileNameMap: prefixFileNames(data, requiredFiles),
-    optionalDescription: '',
```

Comment:
This `optionalDescription` property was removed from `MissingFileOverview`, so we don't need to set it here.